consumers/campaigner/main.go

```diff
@@ -25,12 +25,20 @@ import (
 	"github.com/mailbadger/app/services/campaigns"
 	"github.com/mailbadger/app/services/templates"
 	"github.com/mailbadger/app/storage"
+	"github.com/mailbadger/app/storage/redis"
 	"github.com/mailbadger/app/utils"
 )
 
+// Cache prefix and duration parameters
+const (
+	cachePrefix   = "campaigner:"
+	cacheDuration = 300000 * time.Millisecond
```

Comment:
Can't we set here a more friendly duration? Something like 7 * time.Day. Or if Hour is the highest possible, we'll still need to leave a comment indicating it that it's 7 days or something like that.